### ISSUE TITLE
WA starter kit: Watson discovery actions not loading

### ISSUE THREAD

**Issue by yusufihmirza** [watson-developer-cloud/assistant-toolkit#293]
So I installed watson discovery custom extension from starter kit [here](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/starter-kits/watson-discovery/watson-discovery-query-openapi.json) by import openAPI which went successfully. I added the instance with API key.

Then I was trying to load the actions from [here](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/starter-kits/watson-discovery/watson-discovery-query-actions-generic.json) to which I get the below error:

![image](https://github.com/user-attachments/assets/886d9760-dbee-47cf-9767-666595674ab2)

Error message:
```
Error
can only contain word, underscore, hyphen or dot characters, and must not start with a prefix of "sys" or "system" or "user_defined_".
Transaction id: 5db4b02b-c929-4b5c-9a1b-045bf291f2ae
```

Any idea why is it failing? I am using a new instance of Watson assistant from IBM techzone so assuming it is running on the latest version. Perhaps some issue with version compatibility?

Thanks!

**Comment by jwm4** [comment]
Yes, that kit is quite old, so it sounds like it needs to be updated.

**Comment by yusufihmirza** [comment]
Thanks @jwm4 is there another place where I can find a working watson discovery action reference? also will this kit be updated any time soon to make it usable?

**Comment by leighawilliamson** [comment]
I see that this kit hasn't been fixed in the past 3 months. I'm having this same problem and in my case it's to prepare a customer POC. Is there something I can do to help get this fixed? 
I would edit the action skill JSON directly but it's not clear how many violations of the content rules exist in it. If somebody has the skill already in a watsonx Assistant instance somewhere, could they just export a new version of the JSON?

Or, as [yusufihmirza] asks, does anyone have an action skill integrated with Discovery that they can export and contribute?

**Comment by jwm4** [comment]
There is a more recent one in the second example [here](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/language-model-conversational-search) that also includes an integration with watsonx.ai to generate answers from search results.  I don't know if that one works either, but since it is more recent, it might.  If you don't want the integration with watsonx.ai, it should be pretty easy to start with that one and remove the watsonx.ai / answer generation stuff.

**Comment by leighawilliamson** [comment]
Awesome! Thanks Bill! I'll switch to that repo and see how far I can go. The customer POC is intended to include a RAG summarization of the search results, so the watsonx.ai integration will be useful.
